lib/widget/show_appointment.dart

```diff
@@ -0,0 +1,142 @@
+    final _appointments = await _firebaseService.getAppointments();
+    print('=[]= showAppointment = $_appointments');
+    print('${_appointments.length}');
+    return _appointments;
+  }
+
+  var _userId;
+  var _userCollection;
+  @override
+  void initState() {
+    super.initState();
+    __initData();
+    var userId = _firebaseService.getUserId();
+    setState(() {
+      _userId = userId;
+      _userCollection = _firebaseService.getCollectionSnapshotByDocId(
+          collection: 'Users', docId: _userId);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<DocumentSnapshot>(
+        stream: _userCollection,
+        builder: (context, userCollection) {
+          if (!userCollection.hasData) {
+            return Center(
+              child: Column(
+                children: <Widget>[
+                  CircularProgressIndicator(),
+                  Text('Loading...'),
+                ],
+              ),
+            );
+          } else {
+            return FutureBuilder<Map<String, dynamic>>(
+                future:
+                    _firebaseService.getLatestAnSubCollection(userId: _userId),
+                builder: (context, anSubCollection) {
+                  if (!anSubCollection.hasData) {
+                    return Center(
+                      child: Column(
+                        children: <Widget>[
+                          CircularProgressIndicator(),
+                          Text('Loading...'),
+                        ],
+                      ),
+                    );
+                  }
+                  return ListView(
+                    shrinkWrap: true,
+                    children: <Widget>[
+                      Column(
+                        children: <Widget>[
+                          for (var appointment in appointments)
+                            ExpansionTile(
+                              title: Text(
+                                _calculationService.formatDateToThaiString(
+                                    isBuddhist: true,
+                                    date: appointment['date'].toDate()),
+                                style: TextStyle(color: Color(0xFFC37447)),
+                              ),
+                              children: <Widget>[
+                                Padding(
+                                  padding: const EdgeInsets.only(left: 20),
+                                  child: Column(
+                                    crossAxisAlignment:
+                                        CrossAxisAlignment.stretch,
+                                    children: <Widget>[
+                                      Text(
+                                        'รายละเอียดการนัดหมาย',
+                                        style: Theme.of(context)
+                                            .textTheme
+                                            .bodyText1,
```

Comment:
So, the UI gonna be changed ?

Reply:
yes

Reply:
<img width="337" alt="Screen Shot 2564-02-03 at 10 02 06" src="https://user-images.githubusercontent.com/57941667/106692288-e0a9e680-6606-11eb-9ab4-72bc1afcd67b.png">

appointment will change without hot reload